Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1528,11 +1528,14 @@ std::unique_ptr<RenderStyle> TreeResolver::generatePositionOption(const Position
         return { };
 
     auto resolveFallbackProperties = [&]() -> RefPtr<const StyleProperties> {
-        if (fallback.positionAreaProperties) {
+        if (fallback.positionArea) {
             ASSERT(!fallback.positionTryRuleName);
             ASSERT(fallback.tactics.isEmpty());
-            return fallback.positionAreaProperties;
+
+            auto property = CSSProperty { CSSPropertyPositionArea, Ref { *fallback.positionArea } };
+            return ImmutableStyleProperties::create(std::span { &property, 1 }, HTMLStandardMode);
```

Comment:
Why are we doing this? We were caching ImmutableStyleProperties and now we aren't?